index.html

```diff
@@ -36,65 +105,68 @@ <h2>
-                </div>
-                <div class="program-description">                    
-                    Have the time to share your thoughts and opinions with experts for each topic.                            
-                </div>
-            </div>
-            <div class="program-list">
-                <div class="program-icon">
-                    <img src="assets/images/program-icon-4.png" alt="program icon 4">
+                <div class="program-list">
+                    <div class="program-icon">
+                        <img src="assets/images/program-icon-3.png" alt="program icon 3">
+                    </div>
+                    <div class="program-title">
+                        Forum
+                    </div>
+                    <div class="program-description">                    
+                        Have the time to share your thoughts and opinions with experts for each topic.                            
+                    </div>
                 </div>
-                <div class="program-title">
-                    Workshop
+                <div class="program-list">
+                    <div class="program-icon">
+                        <img src="assets/images/program-icon-4.png" alt="program icon 4">
+                    </div>
+                    <div class="program-title">
+                        Workshop
+                    </div>
+                    <div class="program-description">                    
+                        Try creating your own work using open source license rather than just watching at it.                            
+                    </div>
                 </div>
-                <div class="program-description">                    
-                    Try creating your own work using open source license rather than just watching at it.                            
-                </div>
-            </div>
-            <div class="program-list">
-                <div class="program-icon">
-                    <img src="assets/images/program-icon-5.png" alt="program icon 5">
-                </div>
-                <div class="program-title">
-                    Ignite
-                </div>
-                <div class="program-description">                    
-                    Get opportunities to network with CC affiliates around the world, also after the summit.                            
+                <div class="program-list">
+                    <div class="program-icon">
+                        <img src="assets/images/program-icon-5.png" alt="program icon 5">
+                    </div>
+                    <div class="program-title">
+                        Ignite
+                    </div>
+                    <div class="program-description">                    
+                        Get opportunities to network with CC affiliates around the world, also after the summit.                            
+                    </div>
```

Comment:
- Kindly, Program List is a section, and the program list are article, so let's update it change the `div` tags to `section` and each program list with an `article` tag to make it semantically correct 👍